### DISCUSSION TITLE
How to implement manage attaching and detaching nested relationships?

### DISCUSSION THREAD

**Discussion by devmansurov** [wintercms/winter#1105] [Q&A]
I looked at an [example here](https://octobercms.com/support/article/ob-21) for creating data using nested relationships, but I didn’t understand how to manage the attachment and detachment of nested relationship data (belongsToMany, morphToMany, hasMany).

How to create and make partials correctly in this case?

**Comment by LukeTowers** [comment]
@devmansurov I'm unsure what exactly you're asking.
